build/common/installer/conf/azm-containers-parser-multiline.conf

```diff
@@ -0,0 +1,17 @@
+# [MULTILINE_PARSER]
+#     name          dotnet-regex
+#     type          regex
+#     flush_timeout 1000
+    #
+    # Regex rules for multiline parsing
+    # ---------------------------------
+    #
+    # configuration hints:
+    #
+    #  - first state always has the name: start_state
+    #  - every field in the rule must be inside double quotes
+    #
+    # rules |   state name  | regex pattern                  | next state
+    # ------|---------------|--------------------------------------------
+    # rule      "start_state"    "/^System.*/"                    "cont1"
```

Comment:
is this only covers the exception starts with System.*, right? are there any other scenarios (which  has application specific exceptions) needs to be covered?

Reply:
working on improving this